### DISCUSSION TITLE
ROS Interface in macOS

### DISCUSSION THREAD

**Discussion by fabriziocacicia** [softbankrobotics-research/qibullet#88] [Q&A]
Is there a way to setup the ROS interface in macOS?

I have both qibullet and ROS properly functioning in macOS, but I'm unable to follow the ROS Interface tutorial since it's needed to install some packages using apt-get install.
Is there any other way to make the wrapper works?

**Comment by mbusy** [comment]
Hi, you can probably try to build the required packages from source (end of [this section](https://github.com/softbankrobotics-research/qibullet/wiki/Tutorials:-ROS-Interface#installations)), but I doubt that it will work.
